api/hasura/images.ts

```diff
@@ -0,0 +1,45 @@
+import { VercelResponse } from '@vercel/node';
```

Comment:
This file should be in `api-lib`, not `api`.

Reply:
Same comment as below, api-lib is imported by the FE and it seemed inappropriate to import VercelResponse in there. Perhaps some other folder structure would be helpful here.

Reply:
moving this, i was tripping